packages/charting/controller/src/index.js

```diff
@@ -129,6 +139,9 @@ export const getScore = (question, session, env = {}) => {
           result = 0;
           answers[index].correctness.label = 'incorrect';
         }
+        if (value === corrAnswer.value && lowerCase(label) === lowerCase(corrAnswer.label)) {
```

Comment:
Same here. I would store these equalities in a variable and use their value

Reply:
Done